testing/tests/ui/include-a-folder.stderr

```diff
@@ -0,0 +1,10 @@
+error: unable to open template file '$WORKSPACE/target/tests/trybuild/rinja_testing/templates/a_file_that_is_actually_a_folder.html':
+       Is a directory (os error 21)
```

Comment:
Hum... I think having the error text on the same line might be better. What do you think?

PS: Having a test for this is great, thanks!

Reply:
I wasn't sure if the line does not become too long, then. But chances are that you aren't using such a long path in a real project. :)

Reply:
Boarf, I think it's fine. I find it less confusing this way.